event_model/__init__.py

```diff
@@ -1759,9 +1763,13 @@ def _is_array(checker, instance):
         for name, schema in schemas.items()
     }
 
+if sys.version_info < (3, 8):
+    from importlib_metadata import version  # noqa
+else:
+    from importlib.metadata import version  # noqa
```

Comment:
I'd put this in with the other importlib import at the top of the file

Reply:
Both branches call the same import.  Why is there an `if..else` clause?

Reply:
Removed my comment.  The replacement of `_` by `.` is subtle.